packages/logger/tests/unit/Logger.test.ts

```diff
@@ -159,6 +158,41 @@ describe('Class: Logger', () => {
 
         });
 
+        test('when the Logger\'s log level is SILENT, it DOES NOT print to stdout', () => {
+
+          // Prepare
+          const logger: Logger = createLogger({
+            logLevel: 'SILENT',
+          });
+          const consoleSpy = jest.spyOn(logger['console'], methodOfLogger).mockImplementation();
+
+          // Act
+          logger[methodOfLogger]('foo');
+
+          // Assess
+          expect(consoleSpy).toBeCalledTimes(0);
+        });
+
+        test('when the Logger\'s log level is set through LOG_LEVEL env variable, it DOES print to stdout', () => {
```

Comment:
Added a test that applies to all log methods, when the log level is set through the env var.